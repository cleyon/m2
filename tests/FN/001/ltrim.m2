@readfile FIRST ws.left
Before: FIRST=>@FIRST@<
After: FIRST=>@ltrim FIRST@<
