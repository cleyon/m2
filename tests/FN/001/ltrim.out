Before: FIRST=>     XXX<
After: FIRST=>XXX<
